docs/input-manifest/input-manifest.md

```diff
@@ -256,6 +260,18 @@ Provider spec is further specification build on top of the data from any of the
 
   Zone of the nodepool.
 
+## Autoscaler Configuration
+
+Autoscaler configuration on per nodepool basis. Defines the number of nodes, autoscaler will scale up or down specific nodepool.
```

Comment:
I think it might be better to include information about how the autoscaling works exactly.
What are the thresholds, etc.
Either here, or in `docs/autoscaling`

Reply:
Is there anything specific, you would want to see, other than information that is already in `docs/autoscaling?

Reply:
I added the information about it to the docs, please have a look :)

Reply:
Excellent, thanks Miro